app/views/calculation/gain/valueBeforeLegislationStart.scala.html

```diff
@@ -54,7 +54,8 @@ <h1 class="govuk-heading-xl">@messages("calc.resident.shares.valueBeforeLegislat
         @formInputMoney(
             field = valueBeforeLegislationStartForm("amount"),
             question = messages("calc.resident.shares.valueBeforeLegislationStart.question"),
-            hideLabel = true
+            labelClass = "govuk-label--m",
+            hintText = Some(Messages("calc.resident.shares.valueBeforeLegislationStart.hintText"))
```

Comment:
This hint is missing an example

Reply:
fixed on [ad05c19](https://github.com/hmrc/cgt-calculator-resident-shares-frontend/pull/184/commits/ad05c195e7c914ece2526ab81468c71cd7e46ec5)